candi/cloud-providers/aws/openapi/cluster_configuration.yaml

```
@@ -205,8 +205,6 @@ apiVersions:
         properties:
           associatePublicIPToMasters:
             type: boolean
```

```suggestion
          associatePublicIPToMasters:
            type: boolean
```
I think we remove this code too